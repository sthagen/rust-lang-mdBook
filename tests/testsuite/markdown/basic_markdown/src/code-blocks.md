# Code blocks

```
This is a codeblock
```

```rust
// This links to a playpen
```

```bash,editable
# This is an editable codeblock
```

```text cls1,,cls2	cls3
Text with different classes.
```

    Indented
    code
    block.

```rust,edition2021
let x = 1;
```

```rust
fn main() {
    println!("hello");
}
```
