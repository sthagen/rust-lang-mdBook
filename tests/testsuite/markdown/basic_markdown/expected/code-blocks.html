<h1 id="code-blocks"><a class="header" href="#code-blocks">Code blocks</a></h1>
<pre><code>This is a codeblock
</code></pre>
<pre class="playground"><code class="language-rust"><span class="boring">#![allow(unused)]
</span><span class="boring">fn main() {
</span>// This links to a playpen
<span class="boring">}</span></code></pre>
<pre><code class="language-bash editable"># This is an editable codeblock
</code></pre>
<pre><code class="language-text cls1 cls2 cls3">Text with different classes.
</code></pre>
<pre><code>Indented
code
block.
</code></pre>
<pre class="playground"><code class="language-rust edition2021"><span class="boring">#![allow(unused)]
</span><span class="boring">fn main() {
</span>let x = 1;
<span class="boring">}</span></code></pre>
<pre class="playground"><code class="language-rust">fn main() {
    println!("hello");
}</code></pre>